plugins/plugin_repeat.py

```diff
@@ -13,17 +13,8 @@ def __init__(self,slack_client=None, plugin_config=None):
             slack_client=slack_client,
             plugin_config=plugin_config
         )
-        self.words = [
-            'banana',
-            'bananas',
-            'banana\'s',
-            'apple',
-            'apples',
-            'apple\'s',
-            'pineapple',
-            'pineapples',
-            'pineapple\'s'
-        ]
+        with open("words.txt") as f:
```

Comment:
So like `def load_word_list(file_path): ....`

and then in the init you can do: 
`self.words = load_word_list('./words.txt')`